packages/component-library/src/CivicCard/CivicCardLayoutFull.js

```diff
@@ -326,6 +326,18 @@ function CivicCardLayoutFull({ isLoading, data, cardMeta }) {
             id="authors"
           >
             <h2>Who made this?</h2>
+            {/* temporary implementation */}
+            {cardMeta.authors.length > 4 && (
```

Comment:
Seems a bit weird to us `> 4`...

Reply:
I'll add a comment on that... some cards have `authors: "demo"` and didn't want to apply that to those.

Reply:
Oh.. It's a hacky way to get around arrays vs strings. This should work for now I guess